iati/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from splinter import Browser
+from iati.settings.local import DJANGO_ADMIN_USER, DJANGO_ADMIN_PASS
+
+LOCALHOST = 'http://127.0.0.1:8000/'
+
+
+@pytest.fixture(scope='session', params=[
+    'chrome',
+    'firefox'
+])
+def multibrowser(request):
+    """Create multiple browsers for tests."""
+    return Browser(request.param)
+
+
+@pytest.fixture(scope='function')
+def admin_browser(browser):
+    """Create a browser that is logged in to the CMS."""
+    browser.visit(LOCALHOST+'admin/')
+    browser.fill('username', DJANGO_ADMIN_USER)
```

Comment:
Could we add this set up to the `local.py.example`.

Potentially add a note in the README after the `createsuperuser` step https://github.com/IATI/preview-website/blame/add-page-models/README.md#L41 to add the detail about adding the credentials that were used to make an account to local.py

Reply:
861895a